OrbitQt/orbitmainwindow.cpp

```diff
@@ -1165,11 +1173,38 @@ void OrbitMainWindow::SetTarget(const orbit_qt::StadiaTarget& target) {
   app_->SetGrpcChannel(connection->GetGrpcChannel());
   app_->SetProcessManager(target.GetProcessManager());
   app_->SetTargetProcess(target.GetProcess());
+
   // TODO (b/173011168): Color should not be defined inline. Replace with defined "good color"
   // (green).
-  target_label_->setStyleSheet("#TargetLabel { color: #66BB6A; }");
+  static const QString kTargetLabelDefaultStyleSheet = "#TargetLabel { color: #66BB6A; }";
+  target_label_->setStyleSheet(kTargetLabelDefaultStyleSheet);
   target_label_->setText(QString::fromStdString(target.GetProcess()->name()) + " @ " +
                          target.GetConnection()->GetInstance().display_name);
+
+  using ProcessInfo = orbit_grpc_protos::ProcessInfo;
+  target.GetProcessManager()->SetProcessListUpdateListener([&](std::vector<ProcessInfo> processes) {
```

Comment:
For readability and to allow easier modification later, I'd prefer to provide a `OrbitMainWindow::OnProcessListUpdated` and call this inside `invokeMethod`.

Reply:
Makes sense. I debated that with myself as well!

Reply:
Done.